benchmark/single-source/KeyPathPerformanceTests.swift

```diff
@@ -0,0 +1,1009 @@
+//===--- KeyPathPerformanceTests.swift -------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2022 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+import TestsUtils
+
+public let benchmarks = [
+  BenchmarkInfo(
+    name: "KeyPathNestedStructs",
+    runFunction: run_KeyPathNestedStructs,
+    tags: [.validation, .api],
+    setUpFunction: setupKeyPathNestedStructs
+  ),
+  BenchmarkInfo(
+    name: "KeyPathDirectAccess",
+    runFunction: run_testDirectAccess,
```

Comment:
Looks like this benchmark just serves as the baseline for comparing with the keypath version. But the difference between both benchmarks is not reported in a benchmark run, e.g. on CI. So the only way to see the difference is to compile the benchmarks locally and manually compare both results.
And `KeyPathDirectAccess` as a stand-alone benchmark is not very useful, IMO.

You can keep KeyPathDirectAccess if you think you'd like to do this manual comparison by your own, but then please add the skip tag, so that it does not run by default.
Or you just delete it.

Reply:
I've added the `.skip` tag to retain the benchmark for future reference.